spec/integration_spec.rb

```diff
@@ -35,13 +35,13 @@ def executable(options={})
   end
 
   def ensure_folder(folder)
-    `mkdir -p #{folder}` unless File.exist?(folder)
+    FileUtils.mkpath(folder) unless File.exist?(folder)
   end
 
   def run_tests(test_folder, options={})
     ensure_folder folder
     processes = "-n #{options[:processes]||2}" unless options[:processes] == false
-    command = "cd #{folder} && #{options[:export]} #{executable(options)} #{test_folder} #{processes} #{options[:add]} 2>&1"
+    command = "cd #{folder} && ruby #{options[:export]} #{executable(options)} #{test_folder} #{processes} #{options[:add]} 2>&1"
```

Comment:
that looks wrong ... executable already includes ruby ?

Reply:
if it only works on windows then make it windows opt-in ?
... if the exports don't work also leave them out ... or maybe don't change that line for now so we can get the other fixes merged ...